packages/wdio-appium-service/src/utils.ts

```diff
@@ -25,44 +21,26 @@ export function getFilePath (filePath, defaultFilename) {
     return absolutePath
 }
 
-export function getAppiumCommand (moduleName = 'appium') {
```

Comment:
moved from utils.js as a private member of AppiumLauncher class. It implies to be a part of Appium class and we can avoid additional import of a logger.

Reply:
sounds good to me